pkg/api/metadata.go

```diff
@@ -69,14 +85,20 @@ func (m *Metadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // ServeHTTP sends API version v1 documentation response.
 func (v *VersionMetadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	version, ok := GetCommitSha()
+	if ok != nil {
+		version = ok.Error()
+	}
```

Comment:
Maybe we could send the error instead as a `500` http code error?

Reply:
Should we rename the `ok` return to `err` here? That's the default name for an error returned from a function. `ok` makes it sounds like it's a boolean.

Reply:
Thanks @pb82 @machi1990. Applied the changes!

Reply:
> what do you think about returning 500 from the endpoint if fetching the version failed?

The 500 response is returned as an error in json format so it can parsed. 
Concretely, my suggestion is when an error occurs if to call
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/ce9749f3a293242d72def0349a8e19d67fa65672/pkg/shared/handle_error.go#L11
and return

So the code here will look like

```go
if err != nil {
 shared.HandleError(r,w,err)	
 return
}
```

@pb82 @valeriiashapoval